src/components/PromiseChart/index.js

```diff
@@ -5,33 +5,37 @@ import Share from "@/promisetracker/components/Share";
 
 import useStyles from "./useStyles";
 
-function PromiseChart({ chartLink, title, description }) {
+function PromiseChart({ chartLinks, title, description }) {
   const classes = useStyles();
 
-  const iframeSrc = chartLink || null;
+  const iframeSrcs = chartLinks.trim().split(/\s+/) || null;
```

Comment:
```suggestion
  if (!chartLinks?.length) {
    return null;
  }
  const iframeSrcs = chartLinks.trim().split(/\s+/);
```
From `propTypes` looks like `chartLinks` could be null and we also don't want `iframeSrcs` to be `null` since the `.map` in render below will throw an error.

Reply:
Oh, just pushed a fix for that with this approach https://github.com/CodeForAfrica/PromiseTracker/pull/226/files#diff-1ed2a3d5cdd9cdac6da15ca6ad3c0a7c444eb2f64cd048a8702c4925f5632f5cR15

@kilemensi

Reply:
Though I think I will go with yours as it return early